src/prog_models/utils/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+from .progress_bar import ProgressBar
```

Comment:
```suggestion
from .progress_bar import ProgressBar

```
Missing newline at end of file